pallets/loans/src/lib.rs

```diff
@@ -45,18 +47,15 @@ use sp_runtime::{
 };
 use sp_std::result;
 use sp_std::vec::Vec;
-
-pub use pallet::*;
-
-pub mod weights;
-
 pub use weights::WeightInfo;
 
-#[cfg(test)]
 mod mock;
 mod rate;
-#[cfg(test)]
 mod tests;
+pub mod weights;
+
+pub(crate) const OVERFLOW: DispatchError = DispatchError::Arithmetic(ArithmeticError::Overflow);
+pub(crate) const UNDERFLOW: DispatchError = DispatchError::Arithmetic(ArithmeticError::Overflow);
```

Comment:
we can remove `pub(crate)` if this will only be used by this pallet, if we would like to keep it we can move it to a common place

also this should be
```
 pub(crate) const UNDERFLOW: DispatchError = DispatchError::Arithmetic(ArithmeticError::Underflow);
 ```
right?

Reply:
`ArithmeticError` already implemented the `From` for `DispatchError`. I don't think it's necessary to type the const one.
https://github.com/paritytech/substrate/blob/master/primitives/runtime/src/lib.rs#L593

Reply:
We can use `ArithmeticError` directly like this. https://github.com/AcalaNetwork/Acala/blob/master/modules/loans/src/lib.rs#L325

Reply:
Removed in favor of local declarations